### PR TITLE
Add documentation for MediaPlaceholder props.

### PR DESCRIPTION
**Pull request by jorgefilipecosta**
## Description
This PR just adds documentation for the MediaPlaceholder props.
I noticed during a recent change that the props were not documented.

## How has this been tested?
This PR just affects the documentation.

### CODE REVIEW

File: packages/block-editor/src/components/media-placeholder/README.md [hunk 1]
```diff
@@ -27,6 +27,87 @@ const { MediaPlaceholder } = wp.editor;
 	}
 ```
 
+## Props
+
+### accept
```
Comment (gziolo) [inline packages/block-editor/src/components/media-placeholder/README.md]: Should it be clarified what's the difference between `accept` and `allowedTypes`, if any?

File: packages/block-editor/src/components/media-placeholder/README.md [hunk 2]
```diff
@@ -27,6 +27,87 @@ const { MediaPlaceholder } = wp.editor;
...
+### onError
+
+Callback called when an upload error happens.
+
+- Type: `Function`
+- Required: No
+
+### onSelect
+
+Callback called when the files are selected/uploaded, the selected media are passed as an argument.
```
Comment (gziolo) [inline packages/block-editor/src/components/media-placeholder/README.md]: > the selected media are passed as an argument

In what format?

**Review by gziolo — APPROVED** [review]
It's great to see all those props documented. Thank you for working on it. I left two comments where I would expect to learn more details.

**Comment by jorgefilipecosta** [comment]
Thank you for the review @gziolo your feedback was applied before the merge.

**Pull request merged**